pkg/capacity/manager.go

```diff
@@ -83,15 +105,16 @@ func GetOptimalTargetCluster(preferredCluster, namespace string, getMemberCluste
 		provisioned2 := counts.SpacesPerClusterCounts[optimalTargetClusters[j]]
 		threshold2 := config.CapacityThresholds().MaxNumberOfSpacesSpecificPerMemberCluster()[optimalTargetClusters[j]]
 
-		// Let's round the number of provisioned users down to closest multiple of 50
+		// Let's round the number of provisioned users down to the closest multiple of 50
 		// This is a trick we need to do before comparing the capacity, so we can distribute the users in batches by 50 (if the clusters have the same limit)
-		provisioned1 = (provisioned1 / 50) * 50
-		provisioned2 = (provisioned2 / 50) * 50
+		provisioned1By50 := (provisioned1 / 50) * 50
```

Comment:
(y/x)*x=y because (y/x)*(x/1)=y/1=y therefore provisioned1By50=provisioned1

Reply:
in theory (on paper) yes. But not in Golang when using integers (whole numbers).
eg. 
```
provisoned1 = 108
(provisioned1 / 50) is evaluated as 2
(provisioned1 / 50) * 50 is 100
```
I could change it to 
```
provisioned1 - (provisioned1 % 50)
```
if it would make more sense ( `%` is modulo)